samples/HelloSignalR/Providers/AuthenticationProvider.cs

```diff
@@ -0,0 +1,34 @@
+﻿using AspNet.Security.OpenIdConnect.Server;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Authentication.OpenIdConnect;
+using AspNet.Security.OpenIdConnect.Extensions;
+
+namespace HelloSignalR.Providers {
+    public class AuthenticationProvider : OpenIdConnectServerProvider {
+        public override Task ValidateClientAuthentication(ValidateClientAuthenticationContext context) {
+            if (context.ClientId == "AspNetContribSample") {
+                context.Validated();
```

Comment:
Agreed :smile: 

In a nutshell, `Skipped` (or `Skip` in the nightly builds) is like `Validated` but for clients you can't trust (typically, JS or mobile clients): the request is not rejected, but ASOS is informed that the caller cannot be trusted and that the token is not confidential.